components/SectionPage.vue

```diff
@@ -0,0 +1,328 @@
+                    icon="comment"
+                    :value="getCommentCountById(story.legacyId, moreResultsDisqusData)"
+                    :href="`/${story.ancestry[0].slug}/${story.meta.slug}?to=comments`"
+                  />
+                </template>
+              </article-metadata>
+            </v-card>
+          </div>
+          <div
+            v-if="chunkIndex%2 === 0"
+            class="ad-container"
+          >
+            <hr
+              class="u-border-accent u-hide-until--m u-space--double--bottom"
+              aria-hidden="true"
+            >
+            ad goes here
+            <hr
+              class="u-border-accent u-hide-until--m u-space--double--top"
+              aria-hidden="true"
+            >
+          </div>
+          <div>
+            <v-card
+              v-for="(story, index) in moreResultsChunks[chunkIndex].slice(4,11)"
+              :key="index"
+              class="gothamist u-space--double--bottom mod-large"
+              :show-gallery-icon="hasGallery(story.leadAsset)"
+              :image="getArticleImage(story.leadAsset, story.ancestry[0].slug, story.listingImage)"
+              :image-height="150"
+              :image-width="150"
+              :title="story.title"
+              :title-link="`/${story.ancestry[0].slug}/${story.meta.slug}`"
+              :subtitle="story.description"
+              :tags="story.sponsoredContent ? [{ name: 'sponsored' }] : []"
+            >
+              <article-metadata
+                :publish-date="!story.updatedDate ? fuzzyDateTime(story.meta.firstPublishedAt) : null"
+                :updated-date="story.updatedDate ? fuzzyDateTime(story.updatedDate) : null"
+              >
+                <template
+                  v-if="story.legacyId"
+                  v-slot:comments
+                >
+                  <v-counter
+                    icon="comment"
+                    :value="getCommentCountById(story.legacyId, moreResultsDisqusData)"
+                    :href="`/${story.ancestry[0].slug}/${story.meta.slug}?to=comments`"
+                  />
+                </template>
+              </article-metadata>
+            </v-card>
+          </div>
+        </div>
+        <v-spacer size="double" />
+        <div
+          v-if="moreResults && moreResults.length > 0"
+          class="l-container u-align-center"
+        >
+          <v-button
+            class="more-results"
+            :disabled="!moreResultsLoaded"
+            @click="getMoreResults"
+          >
+            <span v-if="moreResultsLoaded">More Results</span>
+            <span v-if="!moreResultsLoaded">Loading...</span>
+          </v-button>
+        </div>
+      </section>
+      <v-spacer />
+    </div>
+  </div>
+</template>
+
+<script>
+import gtm from '@/mixins/gtm'
+import disqus from '@/mixins/disqus'
+import { mapState } from 'vuex'
+
+const {
+  formatTags,
+  fuzzyDateTime,
+  getArticleImage,
+  hasGallery
```

Comment:
I like this reusable article logic. We don't need it now but at some point it might be nice to organize these things into like article-helpers and date-helpers or something

Reply:
Yes, I tried to make things reusable! I wanted to make the results NUGGETS (lol) reusable but each page had some things that were different :(